website/server/libs/apiMessages.js

```diff
@@ -8,6 +8,8 @@ const messages = {
   guildsOnlyPaginate: 'Only public guilds support pagination.',
   guildsPaginateBooleanString: 'req.query.paginate must be a boolean string.',
   guildsPageInteger: 'req.query.page must be an integer greater than or equal to 0.',
+  groupIdRequired: 'req.params.groupId must contain a groupId.',
```

Comment:
can you remove the strings from the english locale now that they've been added here?